Homework5/1.py

```diff
@@ -0,0 +1,11 @@
+# Напишите программу, удаляющую из текста все слова, в которых присутствуют все буквы "абв".
+
+text = 'фыыфвфыфрваамвб  абв оывпаып ылыабввра ывр  ыввлораывьтлмьбиры лыарпюблрыпв лрвывбла ыа неаы сы ыгнпмокуу ив сыовлоир'
+
+lst_text = text.split()
+
+for i in reversed(range(len(lst_text))):
+    if ("а" in lst_text[i]) and ("б" in lst_text[i]) and ("в" in lst_text[i]):
+        del lst_text[i]
```

Comment:
```suggestion
for i in lst_text[::-1]:
    if "а" in i and "б" in i and "в" in i:
        del i
```
Так ведь значительно проще, не правда ли?